app.js

```diff
@@ -2,18 +2,25 @@ const htmlStandards = require('reshape-standard')
 const cssStandards = require('spike-css-standards')
 const jsStandards = require('spike-js-standards')
 const pageId = require('spike-page-id')
+const sugarml = require('sugarml')
+const sugarss = require('sugarss')
```

Comment:
In the latest version of spike's standards for html and css, we decided to remove sugarml and sugarss as a default. We made this decision to cater to most peoples' use cases. Not everyone is familiar with or prefers the whitespace-based syntax, but some prefer it. We are among those who prefer it, but enforcing our choice on everyone is kind of unreasonable. Additionally, since we are an agency and often build sites that we hand off to clients, we found ourselves having to convert back reasonably frequently in order to maximize the chances that the developers taking over maintenance would be familiar and comfortable with the markup and syntax. Under this logic, we figured we should also be maximizing the chances that spike users would be familiar and comfortable with the markup and syntax as well.

When you start a new spike project with the latest version, it will give you the option to start a project with whitespace-based syntax though, so it's easy to continue using it if you like it (like we do for many projects).